src/renderer/Generics/redesign/Splash/Splash.tsx

```diff
@@ -45,15 +45,15 @@ const Splash = ({ onClickGetStarted }: SplashProps) => {
               i18nKey="genericComponents:TermsAgreement"
               components={[
                 <Linking
-                  url="http://nicenode.xyz/terms.html"
```

Comment:
I just tried accessing http://localhost:8080/terms on my local instance but it doesn't seem to work ☹️ we might need to make a /terms/index.html on the web side ?

Reply:
/terms works on the deploy preview hmm

Reply:
I just modified the web side so that it definitely works now

Reply:
not sure if netlify is doing it but it is adding an extra slash `/terms/` ... I'll test if we can do both